life_expectancy/loading_saving.py

```diff
@@ -1,29 +1,43 @@
+"""
+Module to load and save data
+"""
+
 from pathlib import Path
 
 import pandas as pd
 
+PARENT_PATH = Path(__file__).parent
+FILE_PATH = PARENT_PATH / 'data'
+TSV_FILE_NAME = 'eu_life_expectancy_raw.tsv'
+JSON_FILE_NAME = 'eurostat_life_expect.json'
+TSV_FILE_PATH = FILE_PATH / TSV_FILE_NAME
+JSON_FILE_PATH = FILE_PATH / JSON_FILE_NAME
 
-def load_data(file_path: str) -> pd.DataFrame:
-    """
-    Method used to load data.
-
-    Args:
-        file_path [str]: file path to load data.
 
-    Returns:
-        [pd.DataFrame]: Dataframe to be cleaned.
+class DataLoader:
+    """
+    This class is responsible for verifying the format
+    and assign the right load method
     """
-    data_df = pd.read_csv(file_path, sep='\t')
+    def __init__(self, data_format):
+        self.data_format = data_format
```

Comment:
**Suggestion**: make your class receive a Path object and infer the format from there

```suggestion
    def __init__(self, file_path: pathlib.Path) -> None:
        self.file_path = file_path
        self.data_format = file_path.suffix
```

Reply:
Got it! Thank you!